lib/inspec/reporters/json.rb

```diff
@@ -116,5 +117,12 @@ def profiles
       end
       profiles
     end
+
+    def convert_descriptions(data)
+      return [] if data.nil?
+      results = []
+      data.each { |x| results.push({type: x[0].to_s, data: x[1]} ) }
```

Comment:
For your fellow developers, please use meaningful variable names.  It also helps show your intent and clarity of thought.  Thanks!